encrypted-config-value/src/main/java/com/palantir/config/crypto/algorithm/AesAlgorithm.java

```diff
@@ -40,7 +44,7 @@
     public static final String ALGORITHM_TYPE = "AES";
 
     private static final int GCM_AUTH_TAG_LENGTH = 128;
-    private static final int IV_LENGTH = 256 / Byte.SIZE;
+    private static final int IV_LENGTH = 96 / Byte.SIZE;
```

Comment:
Can we have `TAG_LENGTH` and `IV_LENGTH` use the same units for consistency? Since these are private variables we should be able to change this without breaking anything